yt_spoti_converter_django_application/templates/skeleton.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
I think this file is usually named base.html in djangospeak, not skeleton, but maybe things changed